spec/namespaces/cat.yaml

```diff
@@ -2605,15 +2621,15 @@ components:
     cat.snapshots::query.ignore_unavailable:
       in: query
       name: ignore_unavailable
-      description: If `true`, the response does not include information from unavailable snapshots.
+      description: When `true`, the response does not include information from unavailable snapshots.
       schema:
         type: boolean
         default: false
       style: form
     cat.snapshots::query.master_timeout:
       name: master_timeout
       in: query
-      description: Operation timeout for connection to cluster-manager node.
+      description: A timeout for connection to the cluster manager node.
```

Comment:
"The" timeout? Or, even better, do we mean "The amount of time allowed to establish a connection to the cluster manager node"?